packages/scatterplot/src/ScatterPlotItem.js

```diff
@@ -21,18 +21,33 @@ const ScatterPlotItem = ({
     onMouseMove,
     onMouseLeave,
     onClick,
-}) => (
-    <circle
-        cx={x}
-        cy={y}
-        r={size / 2}
-        fill={color}
-        onMouseEnter={onMouseEnter}
-        onMouseMove={onMouseMove}
-        onMouseLeave={onMouseLeave}
-        onClick={onClick}
-    />
-)
+    symbol,
+    point,
+}) =>
+    symbol ? (
```

Comment:
I guess we could easily add a test for this searching for a matching number of custom symbols.